tools/plugin-commands-self-updater/src/selfUpdate.ts

```diff
@@ -57,6 +58,13 @@ export async function handler (
     return `The currently active ${packageManager.name} v${packageManager.version} is already "latest" and doesn't need an update`
   }
 
+  if (opts.wantedPackageManager && opts.managePackageManagerVersions) {
```

Comment:
Should it check if the wanted package manager is "pnpm" before updating the packageManagerField to be pnpm?

Reply:
we can check but the CLI will fail on an earlier stage in that case